Source/WebKit/UIProcess/API/Cocoa/WKNavigationAction.h

```diff
@@ -27,6 +27,7 @@
 
 #if TARGET_OS_IPHONE
 #import <Foundation/Foundation.h>
```

Comment:
Nit - I _think_ the `Foundation` umbrella import is unnecessary if you add the UIKit umbrella.